CmrUI/Screens/News Feed tab/NewsFeedVC.swift

```diff
@@ -8,12 +8,6 @@
 
```

Comment:
@vmsavka Please change the project structure so that no directories will have spaces in their names.

Reply:
OK, it's simple, no pb